Set Appveyor to use PHP 7.2

photodude opened the pull request:
This makes it so the SQLSVR DLL packages install the correct version to work with PHP 7.2

In #2617 it appeared like there was a limitation preventing testing with PHP 7.2, That was based on an oversite in the appveyor set up vs the PHP packages. This addresses that oversite and allows testing on PHP 7.2

photodude committed:
Use PHP 7.2

Majkl578 reviewed (dismissed):
:+1: 
Maybe we can leave one/two of them to run on 7.1?

photodude commented:
Running on a mixed php 7.1/7.2 setup would require a new environment variable to handle the VC versions.
One or two I would choose the lowest versions for PHP 7.1.

photodude committed:
Modify caching

Following a working cache example https://github.com/morozov/diff-sniffer-core/blob/master/.appveyor.yml

photodude committed:
just C:\tools\php

Majkl578 commented:
Ok, let's just keep 7.2. We test different versions on Travis anyway.

photodude committed:
revert cache change

Ocramius reviewed (approved):
Thanks @photodude! \o/
